app/decorators/storage_system_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class StorageSystemDecorator < MiqDecorator
```

Comment:
The proposed new `StorageSystem` model was replaced with the existing `PhysicalStorage` model wasn't it?

Reply:
removed